src/msg/async/dpdk/DPDK.h

```diff
@@ -672,6 +665,9 @@ class DPDKQueuePair {
   uint8_t _dev_port_idx;
   EventCenter *center;
   uint8_t _qid;
+  clock_type::time_point last_inactive;
```

Comment:
please be consistent when naming member variable. i found that in the rest of this source file, the names of member variables are always started with an underscore (`_`).